ansible_builder/main.py

```diff
@@ -17,8 +18,13 @@
 
 
 class DefinitionError(RuntimeError):
+    sys.tracebacklimit = 0
```

Comment:
What is this doing?

Reply:
It's eliminating the output of traceback before our custom error message prints out.